### DISCUSSION TITLE
A Linux distribution that aims solely starting Qtile

### DISCUSSION THREAD

**Discussion by pannet1** [qtile/qtile#4263] [Q&A]
Hi,
I am using Qtile, Xonsh and Kitty combo on top of EndeavourOS for more now and hope to remain so in the future. Just wondering, if there is a distro that is aimed at only packaging python applications or scripts, so that I can just start a Qtile session with a minimal footprint. I am a python developer by profession and wish to distribute this to the end users. Mainly want to escape from the, but "it works on my computer".

Can you provide some suggestions.

Note: Tinycore looks interesting but should look at removing all WMs (incuding the default one) and then load Qtile on top of it.

**Comment by Sydiepus** [comment]
You can use Arch, gentoo, nixos, debian and freebsd, there's more distros out there i only mentioned a few.

These distros provide a minimal environment (How minimal depends on the distro) that let you install the packages you want.

If by 

> distro that is aimed at only packaging python applications or scripts

you mean a distro purely written in python, the closest thing to that would be gentoo since it's package manager is written in python, for the rest of the system i don't think there's any python replacement for the sound, display, etc..

but all the distros should provide you with python packages that are installable from the official repos and if not you can use pip to install the packages.

**Comment by pannet1** [comment]
if i think now, the question seems very stupid. [this](https://hackaday.com/2020/06/01/python-is-all-youll-ever-need-in-this-linux-distro/) may have furthered my thinking. however, *nix is full of choices. i could very well get a command prompt and install whatever i want on top of it. may be create my own distribution.
anyway thanks a lot for the replies @Sydiepus

**Reply by Sydiepus** [reply]
No worries!
Just to let you know EndeavourOS have an option to install Qtile instead of another DE during the install process, it'll be less time consuming than installing arch or gentoo from scratch since it automatically installs all the required packages.
